draft-ietf-rats-corim.md

```diff
@@ -458,7 +470,23 @@ The following describes each child item of this group.
 
 # CoMID {#sec-comid}
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/7
+A Concise Module Indentifier (CoMID) contains tag that convey information about hardware or firmware component or module used by supply chain entities to provision into a verifier. Each CoMID has a unique tag identity which is used to identify the CoMID unambigously with in the Verifier.
+
+A CoMID defines several types of claims, using triples. Each triple defines a specific type of claim, as given below.
+
+* Reference Triples contain claims for Reference Values that are expected to match the attested Evidence.
```

Comment:
"Claims" vs. "claims"
The structure of a reference triple should be described and should include a reference to the section that defines the triple.

The subsequent list of triples should follow suit.